x-pack/plugins/transform/public/app/sections/transform_management/components/edit_transform_flyout/edit_transform_flyout_form_text_input.tsx

```diff
@@ -9,25 +9,34 @@ import React, { FC } from 'react';
 
 import { EuiFieldText, EuiFormRow } from '@elastic/eui';
 
+import { i18n } from '@kbn/i18n';
+
+import {
+  useEditTransformFlyout,
+  type EditTransformHookTextInputSelectors,
+} from './use_edit_transform_flyout';
+
+function capitalizeFirstLetter(str: string) {
```

Comment:
nit: you can use [capitalize](https://lodash.com/docs/4.17.15#capitalize) from lodash instead

Reply:
Fixed in [a7f74d7](https://github.com/elastic/kibana/pull/149670/commits/a7f74d7fca4ffc695376d51a4902aa1555c590be).

Reply:
FYI I had to revert back to the customized function we were using. For example, for `docsPerSecond`, our function would turn it into `DocsPerSecond` but lodash's `capitalize` would lower case the rest of the string into `Docspersecond`.